modules/getting-started-cli-granting-permissions.adoc

```diff
@@ -6,20 +6,21 @@
 [id="getting-started-cli-granting-permissions_{context}"]
 = Granting view permissions
 
-{product-title} automatically creates a few special service accounts in every project. The default service account takes responsibility for running the pods. {product-title} uses and injects this service account into every pod that launches.
+{product-title} automatically creates several service accounts in every project. The `default` service account takes responsibility for running the pods. {product-title} uses and injects this service account into every pod that launches.
 
-The following procedure creates a `RoleBinding` object for the default `ServiceAccount` object. The service account communicates with the {product-title} API to learn about pods, services, and resources within the project.
+By default, the `default` service account has limited permissions to interact with the OpenShift API.
+
+This procedure assigns the `view` role to the `default` service account to allow it to communicate with the OpenShift API to learn about pods, services, and resources within the project.
```

Comment:
Maybe something like:

As a requirement of this application, this procedure assigns the `view` role to the `default` service account to allow it to communicate with the OpenShift API to learn about pods, services, and resources within the project.